src/utils/helpers/fetchMethods.ts

```diff
@@ -17,6 +17,18 @@ export async function postRequest(body: object, endpointUrl: string, headers: ob
   }
 }
 
-export async function getRequest(body: object, endpointUrl: string) {
-  // TO-DO: Generic GET request
+export async function getRequest(endpointUrl: string, headers: object = {}) {
```

Comment:
i saw your comment in the PR (as in it's not being used), but was wondering, what would be the use case for this ? Wouldn't 'fetch' be supplied to the plugins ?

Reply:
The generic "postRequest" is used in the Login module. I thought I was going to use a GET request here, but didn't end up needing it. Not sure if it'll ever be needed -- we can delete later, perhaps?

Reply:
sure, happy to keep it. My comment was, why we would abstract it further than passing through 'fetch' ? Was it so that authorisation can be set ? (in that case i would assume any type of wrapper around fetch would make fetch work as is but with extra bits added to it, i.e. plugin can still use fetch and set method, headers, etc.. but extra header is added in the wrapper)

Reply:
The reason for abstracting the Login request was so that the large fetch request, with all it's error handling and response processing, etc, could be reduced to just:
```
const loginResult: LoginPayload = await postRequest({ username, password }, loginURL)
```

And I figured it was generic enough that it might be useful elsewhere.